homepage/tests/add_review_search_tests.py

```diff
@@ -59,6 +64,13 @@ def test_get_course_by_name_with_client(client):
 
 
 @pytest.mark.django_db
-def test_wrong_course_name_with_client(client):
+def test_wrong_course_name_with_client(client, sign_in):
     response = client.get('/add_review_search/', {'course': 'Introduction to Buddhism'})
     assert response.status_code == 200
+
+
+@pytest.mark.django_db
+def test_redirect_unsigned_user_to_sign_in_page(client):
+    response = client.get('/add_review_search/', {'course': 'Introduction to Buddhism'})

```

Comment:
Maybe it's better to change the name of the course to a valid one so the test will test only the redirection and won't be affected by other aspects

Reply:
Good call! changed to a valid course name